cli/cli_app.go

```diff
@@ -61,6 +62,14 @@ func CreateTerragruntCli(version string) *cli.App {
    Moreover, for the apply and destroy commands, Terragrunt will first try to acquire a lock using DynamoDB. For
    documentation, see https://github.com/gruntwork-io/terragrunt/.`
 
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "config",
```

Comment:
Do any of the existing Terraform commands expect a `--config` flag? Even if it doesn't now, it may in the future, so to avoid a conflict, we should probably pick a less generic name (e.g. `--terragrunt-config` or `--tc` for short).